README.md

```diff
@@ -899,6 +904,87 @@ gradlew aemInstall -Paem.package.skipDownloadName=false
 ```
 Only matters when Vault properties file is customized then that property could be used to eliminate conflicts.
 
+## Contributing
+
+### Basics
+
+* Work on fork of this repository - In the top-right corner of this page, click Fork.
+* All your changes should be created from and merged into **develop** branch.
+* If you want your changes to be included in upstream release Create Pull Request from your fork **develop** branch into upstream **develop** branch
+    * Before you create Pull Request make sure that plugin is working with **AEM [Single|Multi]-Project Example** projects
+
+### Testing local version of plugin
+
+After you have done some changes make sure that local version plugin works with one of **AEM Project Example**:
+* [AEM Single-Project Example](https://github.com/Cognifide/gradle-aem-single#quickstart) 
+* [AEM Multi-Project Example](https://github.com/Cognifide/gradle-aem-multi#quickstart) 
+
+by publishing your developed version of **Gradle AEM Plugin** to a local maven.
+
+
+Clone into one of **AEM Project Example** projects.
+
+Add local maven *mavenLocal()* on top of repositories in **build.gradle** file:
+
+```groovy
+pluginManagement {
+    repositories {
+        mavenLocal()
+        jcenter()
+        maven { url "https://plugins.gradle.org/m2/" }
+        maven { url  "http://dl.bintray.com/cognifide/maven-public" }
+        maven { url  "https://dl.bintray.com/neva-dev/maven-public" }
+    }
+    ...
+}
+```
+
+Change version of *com.cognifide.gradle:aem-plugin* in **build.gradle** file:
```

Comment:
note that changing version is not needed; will be covered as post release step; thx